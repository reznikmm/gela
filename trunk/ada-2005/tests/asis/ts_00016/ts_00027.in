Associate -I../../../source/asis/spec ts_00XXX/pragma_interface.ada
Action Print Any
Process_Context Library_Unit_Declaration Hello
