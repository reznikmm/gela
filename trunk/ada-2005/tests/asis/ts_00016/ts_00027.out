Declaration_Kind:A_PACKAGE_DECLARATION Hello at pragma_interface.ada [1:1..5:10]
  Defining_Name_Kind:A_DEFINING_IDENTIFIER Hello at pragma_interface.ada [1:9..1:13]
  Declaration_Kind:A_PROCEDURE_DECLARATION V193e at pragma_interface.ada [2:4..2:19]
    Defining_Name_Kind:A_DEFINING_IDENTIFIER V193e at pragma_interface.ada [2:14..2:18]
  Pragma_Kind:AN_UNKNOWN_PRAGMA  at pragma_interface.ada [3:4..3:39]
    Association_Kind:A_PRAGMA_ARGUMENT_ASSOCIATION  at pragma_interface.ada [3:22..3:30]
      Expression_Kind:AN_IDENTIFIER ASSEMBLER at pragma_interface.ada [3:22..3:30]
    Association_Kind:A_PRAGMA_ARGUMENT_ASSOCIATION  at pragma_interface.ada [3:33..3:37]
      Expression_Kind:AN_IDENTIFIER V193e at pragma_interface.ada [3:33..3:37]
  Pragma_Kind:AN_UNKNOWN_PRAGMA  at pragma_interface.ada [4:4..4:49]
    Association_Kind:A_PRAGMA_ARGUMENT_ASSOCIATION  at pragma_interface.ada [4:27..4:31]
      Expression_Kind:AN_IDENTIFIER V193e at pragma_interface.ada [4:27..4:31]
    Association_Kind:A_PRAGMA_ARGUMENT_ASSOCIATION  at pragma_interface.ada [4:34..4:47]
      Expression_Kind:A_STRING_LITERAL "PROCESS_DATA" at pragma_interface.ada [4:34..4:47]
